src/Microsoft.Health.Fhir.Anonymizer.Shared.Core/Processors/GeneralizeProcess.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using EnsureThat;
+using Microsoft.Health.Fhir.Anonymizer.Core.Models;
+using Microsoft.Health.Fhir.Anonymizer.Core.Processors.Settings;
+using Microsoft.Health.Fhir.Anonymizer.Core.AnonymizerConfigurations;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using Hl7.FhirPath;
+using Newtonsoft.Json.Linq;
+using Hl7.Fhir.Model.Primitives;
+
+namespace Microsoft.Health.Fhir.Anonymizer.Core.Processors
+{
+
+    public partial class GeneralizeProcessor : IAnonymizerProcessor
```

Comment:
This file name should be GeneralizeProcessor.cs, same as the class name

Reply:
fixed it